index.d.ts

```diff
@@ -1766,15 +1766,23 @@ declare namespace Shopify {
     processed_at: string;
   }
 
+  interface IInventoryItemCountryHarmonizedSystemCode {
+    harmonized_system_code: string | null;
+    country_code: string | null;
+  }
+
   interface IInventoryItem {
     id: number;
     sku: string;
     tracked: boolean;
     created_at: string;
     updated_at: string;
     requires_shipping: boolean;
-    country_code_of_origin: string;
-    harmonized_system_code: string;
+    cost: string | null;
```

Comment:
```suggestion
    cost: string;
```

Reply:
This can be null if I remember correctly as you're not requested to set the cost per item and can be left blank. I'll look into it tomorrow and change the code if necessary.

Reply:
I have re-checked this and it can be indeed `null` if the cost per item hasn't been set.

Reply:
Ok, thank you.